api/src/main/java/io/opentelemetry/trace/DefaultSpan.java

```diff
@@ -18,33 +18,42 @@
 
 import io.opentelemetry.internal.Utils;
 import java.util.Map;
+import java.util.Random;
 import javax.annotation.concurrent.Immutable;
 
 /**
- * The {@code BlankSpan} is the default {@link Span} that is used when no {@code Span}
+ * The {@code DefaultSpan} is the default {@link Span} that is used when no {@code Span}
  * implementation is available. All operations are no-op except context propagation.
  *
- * <p>When no valid context ({@code null} or {@link SpanContext#BLANK}) is available then
- * implementation propagates {@link BlankSpan#INSTANCE} which encapsulates {@link
- * SpanContext#BLANK}.
- *
  * <p>Used also to stop tracing, see {@link Tracer#withSpan}.
  *
  * @since 0.1.0
  */
 @Immutable
-public final class BlankSpan implements Span {
+public final class DefaultSpan implements Span {
```

Comment:
Hope this will be removed after you resync.

Reply:
Yes, rebased against master.